x-pack/plugins/security_solution/public/management/cypress/tsconfig.json

```diff
@@ -22,6 +22,8 @@
       "path": "../../../tsconfig.json",
       "force": true
     },
-    "@kbn/security-plugin"
+    "@kbn/security-plugin",
+    "@kbn/securitysolution-list-constants",
```

Comment:
Were these changes intentional?

Reply:
I didn't add that, i thought it was auto-added by the build 🤷‍♀️